application/F3DOptionsParser.cxx

```diff
@@ -527,6 +528,11 @@ void ConfigurationOptions::GetOptions(F3DAppOptions& appOptions, f3d::options& o
     f3d::log::error("Error parsing options: ", ex.what());
     throw;
   }
+
+  if (appOptions.ScreenshotFilename.empty())
+  {
+    appOptions.ScreenshotFilename = "{app}/{model}.png";
+  }
```

Comment:
not sure why we would need this ?

Reply:
it's the default value for the CLI argument but I don't know how to set it "properly"

Reply:
can you just set it in the header ?

Reply:
it would be better but it would I don't think `DeclareOption()` can handle it currently https://github.com/f3d-app/f3d/blob/e748359f371a820694609145839f5d856d5ecc01/application/F3DOptionsParser.cxx#L159-L172

Reply:
I'm not following. To me, it should work out of the box as the string case is the simplest one that do not require any code. Also another option is really similar, `verbose`, and is initalized to `info` in the header.

Reply:
You're right, I tried it at the beginning and it didn't work so I blamed `DeclareOption()` and moved on...
turns out with the right parameters it works, my bad